src/app/pages/wallets/wallet-details/pages-wallet-details-item.model.ts

```diff
@@ -25,6 +26,7 @@ export class WalletTransaction {
     this.type = data.type;
     this.amount = data.amount;
     this.walletId = data.walletId;
+    this.categoryId = data.categoryId ?? null;
```

Comment:
`?? null` can be removed, same for the description. in method `create` those should stay because of `Partial<>` used as categoryId won't be there `number | null` but `number | null | undefined`